design/src/App.scss

```diff
@@ -120,6 +120,164 @@ $ribbon-color: #9a0017;
     }
 }
 
+//Form
+.img-overlay {
+  width: 100vw;
+  max-width: 100%;
+  position: absolute;
+  height: 100vh;
+  background-color: rgba(0,0,0,0.5);
+  z-index: 100;
+}
+
+.form-section {
+  width: 100vw;
+  max-width: 100%;
+  position: relative;
+  height: 100vh;
```

Comment:
If there's text content here, consider using `min-height` as a property instead. If someone has a very short browser window, the content will be clipped, which creates an unprofessional look :)